### PR TITLE
[MyMeta] Allow username to be completely removed

### PR DESCRIPTION
**Pull request by dysbulic**
The current code replaces a users name with the name from the backend if the username input is completely cleared.

These changes allow it to be removed and add soft hyphens in a couple places to allow cleaner word splitting on mobile.

Paired with: @jsusim.

### CODE REVIEW

File: packages/web/components/Setup/SetupAvailability.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ export const SetupAvailability: React.FC<SetupAvailabilityProps> = ({
   return (
     <FlexContainer>
       <MetaHeading mb={5} textAlign="center">
-        Availability
+        Avail&#xAD;ability
```
Comment (alalonde) [inline packages/web/components/Setup/SetupAvailability.tsx:68]: why?

**Review by alalonde — COMMENTED** [review]
Do we really want to be able to remove a username?

File: packages/web/components/Setup/SetupAvailability.tsx (hunk 1 above)
Comment (dysbulic) [reply packages/web/components/Setup/SetupAvailability.tsx:68]: It's a soft hyphen that will break on displays that are too narrow.

One of the default mobile devices in Chrome will trigger it: iPhone 4, I think.

Comment (alalonde) [reply packages/web/components/Setup/SetupAvailability.tsx:68]: Ah, interesting

**Comment by alalonde** [comment]
> > Do we really want to be able to remove a username?
> 
> It is that the input can't be cleared to enter a completely new name. When they delete the last character, it puts their old name in the blank.

Ha! That's super annoying! Good find..

**Review by alalonde — APPROVED** [review]

**Pull request merged**